cf_remote/commands.py

```diff
@@ -296,7 +298,21 @@ def _iterate_over_packages(tags=None, version=None, edition=None, download=False
     else:
         for artifact in artifacts:
             if download:
-                download_package(artifact.url, checksum=artifact.checksum)
+                package_path= download_package(artifact.url, checksum=artifact.checksum)
+                if output_dir :
+                    output_dir = os.path.normpath(output_dir)
+                    parent = os.path.dirname(output_dir)
+                    if not os.path.exists(parent):
+                        user_error("'{}' doesn't exist. Make sure this path is correct and exists.".format(parent))
+                    mkdir(output_dir)
+
+                    filename = os.path.basename(artifact.url)
+                    output_path = os.path.join(output_dir, filename)
+                    tmp_filename = '.{}.tmp'.format(filename)
+                    tmp_output_path = os.path.join(output_dir, tmp_filename)
+
+                    shutil.copyfile(package_path, tmp_output_path) 
+                    os.rename(tmp_output_path, output_path)
```

Comment:
@victormlg This logic (generate tmp filename, copy to temp file, rename) is generally useful. Maybe you could put these 4 lines into a new function, called `copy_file` in `utils.py`?